cpp/include/cudf/utilities/default_stream.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+namespace cudf {
+
+/**
+ * @brief Check if the default stream is per-thread.
+ *
+ * @return true if the default stream is per-thread, false otherwise.
+ */
+bool is_per_thread_default_stream();
```

Comment:
What do folks think about `is_ptds_enabled()`?

Reply:
I like it

Reply:
Done.